### ISSUE TITLE
Cannot run visualizer in an angular project. 

### ISSUE THREAD

**Issue by Tallyb** [ui-router/visualizer#8]
Any thoughts? 
Unable to resolve some modules:

  "./ReactElementValidator" in
/D/dev/meteor-angular-socially/node_modules/ui-router-visualizer/release/visualizer.min.js
(web.browser)
  "./ReactDebugTool" in
/D/dev/meteor-angular-socially/node_modules/ui-router-visualizer/release/visualizer.min.js
(web.browser)
  "./ReactDOMDebugTool" in
/D/dev/meteor-angular-socially/node_modules/ui-router-visualizer/release/visualizer.min.js
(web.browser)
  "./ReactComponentTreeDevtool" in
/D/dev/meteor-angular-socially/node_modules/ui-router-visualizer/release/visualizer.min.js
(web.browser)

**Comment by christopherthielen** [comment]
How do you get these errors? In the browser? During bundling? 

How are you including the visualizer in your app? Script tag? Are you re-bundling it using webpack or something? 


---

I'm publishing a UMD module with React inside the bundle.  All the dependencies should be included in the main bundle.

**Comment by Tallyb** [comment]
I am in fact using meteor, so it has its own loader. 
I tried the **html tag** option
index.html:  
`<script src="//npmcdn.com/ui-router-visualizer@2"></script>`
run function: 
```
var vis = window['ui-router-visualizer'];
vis.visualizer(ng1UIRouter);
```
I got the following error in Chrome devtools: 
caught TypeError: Cannot read property 'visualizer' of undefined

trying the **bundler option** (presumably handled by  meteor), and replacing the script tag and var vis with the import, I get the above error during compile. 

I am assuming it has to do with Meteor way of bundling, so I am trying to request some assistance there.
